pkgs/stdenv/generic/setup.sh

```diff
@@ -199,6 +199,29 @@ isELF() {
     if [ "$magic" = $'\177ELF' ]; then return 0; else return 1; fi
 }
 
+# Return success if the specified file is a Mach-O object.
+isMachO() {
+    local fn="$1"
+    local fd
+    local magic
```

Comment:
```suggestion
    local fn="$1" fd magic
```

Reply:
I just took the lines from `isELF`, I didn't want to do it differently. Then both functions should be updated.